src/modules/enrollment/components/dashboardPages/EnrollmentServicesPage.tsx

```diff
@@ -17,7 +17,7 @@ import {
   ServiceFieldsFragment,
 } from '@/types/gqlTypes';
 
-const baseColumns: ColumnDef<ServiceFieldsFragment>[] = [
+export const baseColumns: ColumnDef<ServiceFieldsFragment>[] = [
```

Comment:
can we give this a more descriptive name since its being exported, like SERVICE_COLUMNS

Reply:
Unrelated, but would you mind also combining the category and service type cols? It's too many columns, and sometimes the category and service type are the same. How about combining them like so:

```
  {
    header: 'Service Type',
    render: (service) => {
      const { name, category } = service.serviceType;
      if (name === category) return name;
      return `${category} - ${name}`;
    },
  },
```